GREEDY/Controllers/OCRController.cs

```diff
@@ -1,29 +1,13 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using GREEDY.Models;
-using System.Drawing;
-using Tesseract;
-using Emgu;
-using Emgu.CV;
-using Emgu.CV.Structure;
-using Emgu.CV.OCR;
+﻿using GREEDY.Models;
+using GREEDY.Interfaces;
 
 namespace GREEDY.Controllers
 {
-    public class OCRController
+    public class OCRController : iOCR
     {
-        public Receipt UseOCR(Bitmap image)
+        public Receipt UseOCR(string url)
```

Comment:
What is this url?

Reply:
Yeah could be more specific, I will change this